README.md

```diff
@@ -7,5 +12,8 @@ These instructions assume you know how to use Git, NPM and Bower.
 
 - Clone the repo using `git clone https://github.com/blackbaud/barkbaud-ui.git`
 - Run `npm install`, `bower install`, and `grunt build` from the directory where you cloned the repo
-- Launch the sample site from a web server (the NPM package [http-server](https://www.npmjs.com/package/http-server) makes it easy to launch a web server from any location on your local drive) and navigate to `build/#/artists` to test it out.
+- Launch the sample site from a web server (the NPM package [http-server](https://www.npmjs.com/package/http-server) makes it easy to launch a web server from any location on your local drive; see below).
+    - Type: `npm install http-server -g`.
+    - To run the server on a specific port, type: `http-server -p 5000 -a localhost` (port 8080 should also work).
```

Comment:
I believe localhost/127.0.0.1 will always resolve to the same thing.  I never specify the `-a localhost` option, but always use `http://localhost`